src/components/shared/CallNodeContextMenu.tsx

```diff
@@ -78,6 +79,7 @@ type StateProps = {
   readonly displaySearchfox: boolean;
   readonly browserConnectionStatus: BrowserConnectionStatus;
   readonly innerWindowIDToPageMap: Map<InnerWindowID, Page> | null;
+  readonly sources: SourceTable;
```

Comment:
Wait, this component already has a `thread` prop. Shouldn't we put the sources into the derived thread, the same way we do with the stringTable?